.github/workflows/pull-request.yml

```diff
@@ -37,22 +37,6 @@ jobs:
           profile: minimal
```

Comment:
Is .github/workflows/pull-request.yml composable?  ie, can we have one in each subdirectory and have those included individually by the main `.github/workflows/pull-request.yml`?

Reply:
> You must store workflow files in the .github/workflows directory of your repository.

https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#about-yaml-syntax-for-workflows

We can split up workflows into separate files but right now at least the JavaScript tests rely on the programs being built in the same workflow.